### DISCUSSION TITLE
Exiting ssh session disables key combination

### DISCUSSION THREAD

**Discussion by mortenivar** [kovidgoyal/kitty#6489] [Q&A]
I do not report this as an issue, since I use Kitty 0.28.1 but rather just as something to possibly look into. I enter an ssh session
with kitty +kitten ssh <host>. Usually the key combination ctrl + space works in an non ssh session. This key combination emits "^@". Both in an ssh session and after having exited the ssh session, this key combination is deactivated - it emits nothing.
This may apply to other key combinations as well. The $TERM variable both in and out of ssh is xterm-kitty.

**Comment by kovidgoyal** [comment]
Does not reproduce for me

kitten ssh somemachine
kitten show-key 
Press ctrl+space and see the output:
UNIX		send_text
^@		\x00

**Comment by mortenivar** [comment]
I also get that within the ssh session when using the kitten
show-key command. However, with ctrl-v followed by key
combination, I get no output. But the worst problem is that
when returning from the ssh session, the key combination no
longer works. I need to restart kitty to get it back. I use
ctrl+space to initiate marking of text in my editor. Maybe you
could try to verify that the key combination works in practice
_after_ returning from the ssh session? If it does work for you
but not for me, then what should I be looking at in my own setup?

Thanks, Morten

**Comment by kovidgoyal** [comment]
Yes, it works. And you dont need to restart kitty, you could just do
reset. Almost certainly something on the machine you are connecting to is turning on the kitty keyboard protocol and then either not turning it off when done or not handling the encoding correctly.

**Comment by mehalter** [comment]
We you ever able to solve this @mortenivar ? I'm also having problems specifically with `"^ "` mappings

**Reply by mehalter** [reply]
this should be fixed in the latest commit on kitty with #6780 :)
